### PR TITLE
feat: PD-16825 Add and example for dateComparison operator

### PR DESCRIPTION
**Pull request by josh-wu**
### Issue Link:

PD-16825

### What does it do?

Add and example for dateComparison operator

#### Checklist

- [ ] BREAKING CHANGE - I have/intend to update all places where this package is used.
- [x] I will squash: PR title conforms to standard commit message format (`feat: XX-1234 ...`)
- [ ] I will merge without squashing: All commits conforms to standard commit message format (`feat: XX-1234 ...`)
- [ ] README update

---

#### Screenshots

image

#### Related PRs

* https://github.com/cloudconformity/util-custom-rules-engine/pull/27

### CODE REVIEW

**Comment by patty-chang** [comment]
One minor item and LGTM 👍
image

**Review by patty-chang — APPROVED** [review]

File: AWS/AccessKeyRotatedWithin180days.json [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  },
+  "resource": {
+    "accountId": "933d1234-1234-1234-1234-1234ce55ce62",
+    "organisationId": "12345234-6789-6789-6789-2345ce557890",
+    "resourceId": "1234123412349999234",
+    "service": "IAM",
+    "ccrn": "ccrn:aws:933d1234-1234-1234-1234-1234ce55ce62:IAM:global:user-1234123412349999234",
+    "region": "global",
+    "descriptorType": "iam-user",
+    "data": {
```
Comment (patty-chang) [inline AWS/AccessKeyRotatedWithin180days.json:49]: @josh-wu 
There is no resource.data.resourceId so the dry run result message will have undefined string, so maybe also add the resourceId as well
image

Comment (josh-wu) [reply AWS/AccessKeyRotatedWithin180days.json:49]: Nice catch 🚀

**Comment by josh-wu** [comment]
Address comment
- Add `data.resourceId`
image

**Pull request merged**